lib/db.js

```diff
@@ -439,6 +445,9 @@ module.exports = (
     return this.pool.get('/sessionToken/' + id + '/device')
     .then(
       function (data) {
+        if (! data.deviceId) {
+          data.deviceId = null
```

Comment:
`deviceId` is explicitly nulled here so that the session token object can distinguish between data we're sure has no `deviceId` and data we're unsure about because it originated from a db endpoint that doesn't return device info.

If we were to take the route of always initialising session tokens with device info (see below), we could get rid of this assignment completely. If we don't take that route and this assignment sticks around, I'll add a nice descriptive comment to clarify clear why it's here.